Tracker/Controllers/Trackers/CreatTracker/Controller/EditCategoryViewController.swift

```diff
@@ -4,6 +4,12 @@ final class EditCategoryViewController: UIViewController {
 
     private var editCategoryView: EditCategoryView!
     private var editCategoryText: String?
+    private var indexPathEditCategory: IndexPath?
+    
+    private let categoryStory = TrackerCategoryStore()
+    private var typeEditeCategory: EditCategory?
+    
+    var callBack:((TrackerCategoryCoreData?)->())?
```

Comment:
Я не очень приветствую такой нейминг, вот захочется тебе разные колбеки с экрана кидать, как будешь из называть?
```suggestion
    var callBack: ((TrackerCategoryCoreData?)->())?
```